content/pages/datamanagement/demo.rst

```diff
@@ -91,24 +118,19 @@ Conversion to BIDS
 In order to get a BIDS dataset from the raw dataset, create a new dataset and
 set it up to become a BIDS dataset::
 
-  datalad create bids
-  cd bids
-  datalad run-procedure setup_bids_dataset
+  % datalad create bids
+  % cd bids
+  % datalad run-procedure setup_bids_dataset
 
 Now, install input data as a subdataset::
 
-  datalad install --dataset . --source ../study_ds sourcedata
-  datalad install sourcedata/code/toolbox
-
-
-If the specification wasn't altered, the actual conversion is done by::
-
-  datalad hirni-spec2bids sourcedata/02/studyspec.json
+  % datalad install --dataset . --source ../my_raw_dataset sourcedata --recursive
 
-Note, that this command takes a list of specification files (each expected to be
-an acquisition specification) and converts the respective acquisitions. If you
-want to convert a dataset with multiple acquisitions at once, just use::
+The actual conversion is based on the specification files in the study dataset. You can convert a single one of them (meaning: Everything such a file specifies) or an arbitrary number, including everything at once, of course::
 
-  datalad hirni-spec2bids sourcedata/*/studyspec.json
+  % datalad hirni-spec2bids --anonymize sourcedata/**/studyspec.json
 
+The `anonymize` switch will cause the command to use the anonymized subject identifiers and encode all records of where exactly the data came from into hidden sidecar files, that can tha be excluded from publishing/sharing this dataset.
 
+If you use the BIDS-Validator (https://bids-standard.github.io/bids-validator/) to check the resulting dataset, there should be an error message, though. This is because our events.tsv file references stimuli files, we don't actually have available to add to the dataset.
+For the purpose of this demo, this should be fine.
```

Comment:
Conclude with a summary on what is now possible with this converted dataset. Compatible with BIDS-Apps, could have DICOM metadata aggregated into ...

Reply:
Added such a paragraf, but you might want things to be added ...